bob-common/src/configs/cluster.rs

```diff
@@ -423,6 +423,49 @@ impl Cluster {
             Ok(config)
         }
     }
+
+    pub fn get_testmode(path: String, addresses: Vec<String>) -> AnyResult<Self> {
+        let disks = vec![DiskPath::new("disk_0".to_string(), path)];
+        let len = addresses.len();
+        let mut nodes = Vec::with_capacity(len);
+        let mut vdisk = VDisk::new(0);
+        for (address, i) in addresses.into_iter().zip(0..len) {
+            let node = Node {
+                name: format!("node_{i}"),
+                address,
+                disks: disks.clone()
+            };
+            let replica = Replica::new(node.name().to_string(), node.disks()[0].name().to_string());
+            vdisk.push_replica(replica);
```

Comment:
In test mode cluster should work with multiple shards and without any replication. You should create new vdisk for every node